src/snapred/backend/data/DataFactoryService.py

```diff
@@ -60,30 +60,11 @@ def getSampleFilePaths(self):
     def getGroupingMap(self, runId: str):
         return self.lookupService.readGroupingMap(runId)
 
-    ##### REDUCTION METHODS #####
-
-    @validate_arguments
-    def getReductionState(self, runId: str, useLiteMode: bool) -> ReductionState:
-        reductionState: ReductionState
-
-        if runId in self.cache:
-            reductionState = self.cache[runId]
-
-        else:
-            # lookup and package data
-            reductionState = ReductionState(
-                instrumentConfig=self.getInstrumentConfig(runId),
-                stateConfig=self.getStateConfig(runId, useLiteMode),
-            )
-            self.cache[runId] = reductionState
-
-        return reductionState
-
```

Comment:
This was not deleted, it was moved to below with the other reduction methods.  I'm not sure how we got two reduction methods sections.